lms/static/js/learner_dashboard/views/course_card_view.js

```diff
@@ -23,8 +25,13 @@
 
                 tpl: HtmlUtils.template(pageTpl),
 
-                initialize: function() {
+                initialize: function(options) {
+                    this.enrollModel = new EnrollModel();
+                    if(options.context && options.context.urls){
+                        this.urlModel = new Backbone.Model(options.context.urls);
+                    }
                     this.render();
+                    this.listenTo(this.model, 'change', this.render);
```

Comment:
All change is fine I think. There are two possible actions impacting the rendering of the card:
1. The run selection dropdown
2. The enroll button click.
The 2 actions above would alter different fields in the model, therefore it's easier to render if any fields in the model change.